libs/dh/core/shell/src/lib/shell/shell.component.html

```diff
@@ -15,9 +15,18 @@
 limitations under the License.
 -->
 <watt-shell>
-  <ng-container watt-shell-sidenav> Sidenav </ng-container>
+  <ng-container watt-shell-sidenav>
+    <div class="sidenav">
```

Comment:
issue(non-blocking): The exact same class is used in `watt-shell` component as well. This is not a problem right now because that component is encapsulated but it might become an issue if we remove the encapsulation. Maybe we should use a `watt-` prefix for classes used in Watt just like Material uses `mat-`. What do you think?

Reply:
Good point, I think that we should prefix watt